bindings/pyroot/src/Pythonize.cxx

```diff
@@ -2262,6 +2263,18 @@ namespace {
       return BindCppObject( addr, (Cppyy::TCppType_t)Cppyy::GetScope( "TObject" ), kFALSE );
    }
 
+   //- Pretty printing with cling::PrintValue
+   template <typename T>
+   PyObject *ClingPrintValue(ObjectProxy *self)
+   {
+      auto *cobj = (T*)(self->GetObject());
+      std::string name = "std::vector<float>"; // TODO: How to infer this name for non-ROOT objects?
```

Comment:
@dpiparo @Axel-Naumann Do you know how I can infere the name of the type here? I need it for the call to the interpreter. Unfortunately it seems that `char*` and `std::string` are not valid template parameters and I cannot change the function signature...

Reply:
Btw, besides the missing name of the type, everything works fine!